docusaurus/docs/adding-bootstrap.md

```diff
@@ -3,36 +3,28 @@ id: adding-bootstrap
 title: Adding Bootstrap
 ---
 
-You don’t have to use [reactstrap](https://reactstrap.github.io/) together with React but it is a popular library for integrating Bootstrap with React apps. If you need it, you can integrate it with Create React App by following these steps:
+While you don’t have to use any specific library to integrate Bootstrap with React apps, it's often easier than trying to wrap the Bootstrap jQuery plugins. [React Bootstrap](https://react-bootstrap.netlify.com/) is the most popular option, that strives for complete parity with bootstrap. [reactstrap](https://reactstrap.github.io/) is also a good choice for projects looking for smaller builds at the expense of less features.
```

Comment:
The end of this sentence doesn't sound right. `expense of less featues` sounds kind of like a double negative. Maybe something like `at the expense of some features`?